backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfWriter.java

```diff
@@ -334,6 +337,36 @@ protected void writeGeneralData() {
         _writer.writeElement(CPU_PINNING_POLICY, String.valueOf(vmBase.getCpuPinningPolicy().getValue()));
     }
 
+    private boolean isMdevTypeRequired() {
+        final Version version = vmBase.getCustomCompatibilityVersion() != null ? vmBase.getCustomCompatibilityVersion()
+                : vmBase.getClusterCompatibilityVersionOrigin();
```

Comment:
IIRC cluster-compatiblity-version-origin was only set when reading the ovf (and then we should rather check `version` here)

Reply:
Indeed, fixed, thanks for catching this.